spec/factories/wimi.rb

```diff
@@ -0,0 +1,16 @@
+FactoryGirl.define do
+  factory :wimi, class: 'User' do
+    first_name          'Jim'
+    last_name           'Doe'
+    sequence(:email)    { |n| 'person#{n}@example.com' }
+    chair               @chair
+
+    before(:create) do
+      @chair = Chair.new(name: 'Another Test chair')
```

Comment:
Könntet ihr nicht mit FactoryGirl eine Instanz erstellen? Das würde dann konsistent und in der Test-Datenbank bleiben.